semconv/src/main/java/io/opentelemetry/semconv/ResourceAttributes.java

```diff
@@ -25,7 +25,12 @@ public final class ResourceAttributes {
   /** The URL of the OpenTelemetry schema for these keys and values. */
   @Deprecated public static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.23.1";
 
-  /** The cloud account ID the resource is assigned to. */
+  /**
+   * The cloud account ID the resource is assigned to.
+   *
+   * @deprecated deprecated in favor of stable {@see
+   *     io.opentelemetry.semconv.incubating.CloudIncubatingAttributes#CLOUD_ACCOUNT_ID} attribute.
```

Comment:
(the incubating attributes aren't stable)
```suggestion
   * @deprecated deprecated in favor of {@see
   *     io.opentelemetry.semconv.incubating.CloudIncubatingAttributes#CLOUD_ACCOUNT_ID} attribute.
```